doc/shared/sunstepper/SUNStepper_Description.rst

```diff
@@ -244,16 +280,6 @@ determined by the "consumer" of the :c:type:`SUNStepper`.
    :return: A :c:type:`SUNErrCode` indicating success or failure.
 
 
-.. c:function:: SUNErrCode SUNStepper_SetOneStepFn(SUNStepper stepper, SUNStepperOneStepFn fn)
```

Comment:
Since the `OneStep` function was added back, this documentation needs to be added back as well